src/test/java/io/supertokens/test/FeatureFlagTest.java

```diff
@@ -741,4 +745,283 @@ public void testNetworkCallIsMadeInCoreInit() throws Exception {
+
+                assertEquals(signInResponse.get("status").getAsString(), "OK");
+            }
+        }
+
+        { // tenant 2
+            {
+                JsonObject responseBody = new JsonObject();
+                responseBody.addProperty("email", "test1@example.com");
+                responseBody.addProperty("password", "password");
+
+                JsonObject signInResponse = HttpRequestForTesting.sendJsonPOSTRequest(process.getProcess(), "",
+                        "http://localhost:3567/t1/recipe/signin", responseBody, 1000, 1000, null, SemVer.v4_0.get(),
+                        "emailpassword");
+
+                assertEquals(signInResponse.get("status").getAsString(), "OK");
+            }
+            {
+                JsonObject responseBody = new JsonObject();
+                responseBody.addProperty("email", "test2@example.com");
+                responseBody.addProperty("password", "password");
+
+                JsonObject signInResponse = HttpRequestForTesting.sendJsonPOSTRequest(process.getProcess(), "",
+                        "http://localhost:3567/t1/recipe/signin", responseBody, 1000, 1000, null, SemVer.v4_0.get(),
+                        "emailpassword");
+
+                assertEquals(signInResponse.get("status").getAsString(), "OK");
+            }
+            {
+                JsonObject responseBody = new JsonObject();
+                responseBody.addProperty("email", "test3@example.com");
+                responseBody.addProperty("password", "password");
+
+                JsonObject signInResponse = HttpRequestForTesting.sendJsonPOSTRequest(process.getProcess(), "",
+                        "http://localhost:3567/t1/recipe/signin", responseBody, 1000, 1000, null, SemVer.v4_0.get(),
+                        "emailpassword");
+
+                assertEquals(signInResponse.get("status").getAsString(), "OK");
+            }
+        }
+
+
+        {
+            JsonObject response = HttpRequestForTesting.sendGETRequest(process.getProcess(), "",
+                    "http://127.0.0.1:3567/ee/featureflag",
+                    null, 1000, 1000, null, WebserverAPI.getLatestCDIVersion().get(), "");
+            Assert.assertEquals("OK", response.get("status").getAsString());
+
+            JsonObject alStats = response.get("usageStats").getAsJsonObject().get("account_linking").getAsJsonObject();
+            assertNotNull(alStats);
+
+            assertEquals(3, alStats.entrySet().size());
+            assertTrue(alStats.get("usesAccountLinking").getAsBoolean());
+            assertEquals(2, alStats.get("totalUserCountWithMoreThanOneLoginMethod").getAsInt());
+
+            JsonArray maus = alStats.get("mauWithMoreThanOneLoginMethod").getAsJsonArray();
+            assertEquals(30, maus.size());
+            assertEquals(2, maus.get(0).getAsInt());
+        }
+
+        process.kill();
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STOPPED));
+    }
 }
```

Comment:
modify existing test where there is only multi tenancy and no acc linking, and check that acc linking prop is not being added